ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/ByteCodeBuilder.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright Besu Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.core;
+
+import org.hyperledger.besu.datatypes.Address;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.units.bigints.UInt256;
+
+public class ByteCodeBuilder {
+
+  public enum Operation {
+    ADD("01"),
+    CALL("f1"),
+    CALLDATALOAD("35"),
+    CALLER("33"),
+    DELEGATECALL("f4"),
+    EQ("14"),
+    JUMPDEST("5b"),
+    JUMPI("57"),
+    MLOAD("51"),
+    MSTORE("52"),
+    PUSH("60"),
+    RETURN("f3"),
+    RETURNDATACOPY("3e"),
+    REVERT("fd"),
+    STATICCALL("fa"),
+    TLOAD("b3"),
+    TSTORE("b4");
+
+    private final String value;
+
+    Operation(final String value) {
+      this.value = value;
+    }
+
+    @Override
+    public String toString() {
+      return value;
+    }
+  }
+
+  StringBuilder byteCode = new StringBuilder("0x");
```

Comment:
You might want to use a MutableBytes object instead of a String. You can add to it and have it output to String at the end.

Reply:
I'm leaning toward leaving as a StringBuilder for the following reasons:
1. Opcodes are easier to manage as hex strings since they're more often documented as such ([ex](https://github.com/crytic/evm-opcodes)). It just saves the additional step of having to convert it.
2. `TestCodeExecutor.executeCode()` accepts a hex string and is used elsewhere in the codebase. 
3. There's no real efficiency gain of using `MutableBytes` over `StringBuilder` (correct me if I'm wrong), esp without knowing the number of operations in advance.

Reply:
It's tests, efficiency is moot - just looking at what level of string manipulations you have to do. Anyways, just for testing, so it can go as is.